tests/layer_tests/mo_python_api_tests/test_mo_convert_pytorch.py

```diff
@@ -681,6 +682,134 @@ def create_pytorch_module_with_optional_inputs_case5(tmp_dir):
     return net, ref_model, {"input": ["x", "z"], "input_shape": [[1, 3, -1, -1], [1, 3, -1, -1]]}
 
 
+def create_pytorch_module_with_nested_inputs(tmp_dir):
+    class PTModel(torch.nn.Module):
+        
+        def forward(self, z:Tuple[torch.Tensor, torch.Tensor]):
+            z1, z2 = z
+            zeros = torch.zeros((1, 1))
+            return torch.cat([z1, zeros], 1), torch.cat([z2, zeros], 1) 
+
+    net = PTModel()
+    constant_zeros = ov.opset10.constant(np.zeros((1, 1), dtype=np.float32), dtype=np.float32)
+    shape = PartialShape([1, -1])
+    param1 = ov.opset10.parameter(shape, dtype=np.float32)
+    param2 = ov.opset10.parameter(shape, dtype=np.float32)
+    concat1 = ov.opset10.concat([param1, constant_zeros], 1)
+    concat2 = ov.opset10.concat([param2, constant_zeros], 1)
+    ref_model = Model([concat1, concat2], [param1, param2], "test")
+    return net, ref_model, {"example_input": {"z": (torch.zeros((1, 10)), torch.ones((1, 10)))}}
```

Comment:
I think it makes sense to check the order of outputs. So may be it is better to have inputs and outputs of different shapes?

Reply:
updated